tests/utils/tmpnet.go

```diff
@@ -15,8 +15,9 @@ import (
 )
 
 var DefaultChainConfig = tmpnet.FlagsMap{
-	"log-level":        "debug",
-	"warp-api-enabled": true,
+	"log-level":         "debug",
+	"warp-api-enabled":  true,
+	"local-txs-enabled": true,
```

Comment:
should we enable this only in the load tests?

Reply:
I think it makes sense to enable them for warp tests as well (since it has a load test in warp tests)